docs/.gitignore

```diff
@@ -1,16 +1,16 @@
 .hugo_build.lock
 assets/doc-models/*.json
 bin/
-content/docs/apis
+content/docs/api
 node_modules/
 public/
 scripts/api-documenter/temp
 themes/thxvscode/assets/js/node_modules/
 
 # Generated by build-api-nav during docs build.
-data/allAPIs.yaml
-data/displayNameToPackageName.yaml
-data/packageNameToDisplayName.yaml
+data/*/allAPIs.json
```

Comment:
The original idea was to put packages.json inside these v1, v2 folders, so I did this so we dont delete packages.json within v1 or v2, since we didnt continue with this design change, we can remove these three lines for a generic `data/v*/`

Reply:
Make sense. Updated!